### PR TITLE
CPDB fisa enhancements

### PR DESCRIPTION
**Pull request by fvankrieken**
#102 #289 

I'd go by commit for review. Commits 4, 5, 6 touch a fair amount of the same stuff, but still wanted to group it somewhat logically. Really, commits 5 and 6 are the big ones pertaining to the changes to build logic (5 more of a refactor, 6 adding the new data). 1-3 are all focused on data-library

## Commits
1. update endpoint for facilities in data library
2. add fisa daily budget data to data library
3. edit fisa capital commitment plan in data library to handle the raw files we get from FISA (relied on converting it to a csv before)
4. rename cpdb sql files to make a bit more sense
5. refactor cpdb sql to get rid of some redundancies, name intermediate tables by source, update column names to what we want moving forward. For within the build, tables are named to reflect their source
6. add in new fisa daily budget data to cpdb. This includes some intermediate tables, as well as the new "cpdb_projects" output which effectively replaces the old "cpdb_projects_combined". The old "cpdb_projects" csv is dropped entirely
7. use dcpy recipe logic to build cpdb. This was my first time really using it! Super clean and easy. Just put version in the recipe for now ala pluto, will try to address that in #413 where I'm touching some version stuff

### CODE REVIEW

File: products/cpdb/bash/02_build_attribute.sh [hunk 1]
```diff
@@ -152,3 +153,6 @@ echo 'Removing bad geometries'
 run_sql_command "DROP TABLE IF EXISTS cpdb_badgeoms; CREATE TABLE cpdb_badgeoms (maprojid text);"
 run_sql_command "\COPY cpdb_badgeoms FROM './data/cpdb_geomsremove.csv' DELIMITER ',' CSV;"
 run_sql_file sql/attributes_badgeoms.sql	
+
+# create final table
+run_sql_file sql/projects_combined.sql -v ccp_v=$ccp_v
```
Comment (fvankrieken) [inline products/cpdb/bash/02_build_attribute.sh:158]: This was happening in export step before

**Comment by fvankrieken** [comment]
Okay actually done pushing changes

File: products/cpdb/bash/01_preprocessing.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+source bash/config.sh
+
+run_sql_file "sql/_create.sql"
+
+echo "fixing dot_bridges"
+run_sql_command "ALTER TABLE dot_projects_bridges RENAME COLUMN fmsid TO fms_id;"
+python3 python/dot_bridges.py
+
+echo "fixing doitt_buildingfootprints"
+run_sql_command "ALTER TABLE doitt_buildingfootprints RENAME TO doitt_buildingfootprints_source;"
```
Comment (alexrichey) [inline products/cpdb/bash/01_preprocessing.sh:11]: Fyi, you can just add an "import_as: doitt_buildingfootprints_source" to the recipes

**Review by alexrichey — APPROVED** [review]
🎉🎉🎉

**Pull request merged**